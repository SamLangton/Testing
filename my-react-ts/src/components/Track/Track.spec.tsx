/*
	Snapshot tests for "./Track.tsx"
*/
import * as React from 'react';
import Track from './Track';
import * as renderer from 'react-test-renderer';

describe('./Track.tsx', () => {
	it('should render as expected', () => {
		const component = renderer.create(
			<Track />
		);
		let tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});