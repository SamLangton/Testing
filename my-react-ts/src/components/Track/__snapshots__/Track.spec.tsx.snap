// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`./Track.tsx should render as expected 1`] = `
<div
  className="Track"
/>
`;
